src/js/components/homepage/features/PaneFeature.jsx

```diff
@@ -26,54 +28,90 @@ const PaneFeature = () => {
             <div className="feature-pane__wrapper">
                 <h2 className="feature-pane__title">Featured Content</h2>
```

Comment:
We need to change this to Other Data Act Content

```suggestion
                <h2 className="feature-pane__title">OTHER DATA ACT CONTENT</h2>
```